roles/mongodb_config/tasks/main.yml

```diff
@@ -1,5 +1,10 @@
 ---
 # tasks file for mongodb_config
+- name: Are we in docker?
+  stat:
+    path: is_docker.txt
```

Comment:
Is this only included for the integration tests? I've never seen is_docker.txt except for in this collection.

We might be able to use a fact instead:
`ansible_facts.virtualization_role=='guest' and ansible_facts.virtualization_type=='docker'`

If we need to detect additional container types:
`ansible_facts.virtualization_role=='guest' and ansible_facts.virtualization_type in ['docker', 'lxc', 'podman', 'openvz', 'container']`

Reply:
Yes, I create the is_docker files in the prepare.yml playbooks. That's much better but I'll do that in a separate pull as it's used across a bunch of roles.

I guess this fact alone would do...

`ansible_facts.virtualization_type=='docker'`

Reply:
Yes, a separate PR makes sense. And `ansible_facts.virtualization_type=='docker'` should be fine for a few ansible versions.

Virtualization facts are getting refactored in ansible devel, so I wouldn't be surprised if they added a check for a docker host at some point. The refactoring should be backwards compatible, but a future ansible version will support reporting all supported host and guest virtualization types. Using the new virtualization vars in devel, this check would look like: `'docker' in ansible_facts.virtualization_tech_guest`.